File: sdks/php/src/Plans/Types/CreatePlanDtoBillingInterval.php
<?php

namespace NovaBilling\Plans\Types;

enum CreatePlanDtoBillingInterval: string
{
    case Hourly = "HOURLY";
    case Daily = "DAILY";
    case Weekly = "WEEKLY";
    case Monthly = "MONTHLY";
    case Quarterly = "QUARTERLY";
    case Yearly = "YEARLY";
}

File: sdks/ruby/lib/novabilling/plans/types/create_plan_dto_billing_interval.rb
# frozen_string_literal: true

module Novabilling
  module Plans
    module Types
      module CreatePlanDtoBillingInterval
        extend Novabilling::Internal::Types::Enum

        HOURLY = "HOURLY"
        DAILY = "DAILY"
        WEEKLY = "WEEKLY"
        MONTHLY = "MONTHLY"
        QUARTERLY = "QUARTERLY"
        YEARLY = "YEARLY"
      end
    end
  end
end
